.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+Please include a summary of what the PR does adding relevant motivation and context.
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Trello Ticket ID
+
+Please add a link to the Trello ticket for the task.
+
+## How Can This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test scenario/configuration
+
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] My changes generate no new warnings
+- [ ] Any dependent changes have been merged and published in downstream modules
+
+# Screenshots
```

Comment:
@ttcollins I don't think the screenshots section applies for the backend PRs

Reply:
Ohhh, sorry about, totally skipped my mind